workspace/__lib__/xod-dev/st7735-display/init-b/patch.cpp

```diff
@@ -0,0 +1,18 @@
+
+struct State {};
+
+// clang-format off
+{{ GENERATED_CODE }}
+// clang-format on
+
+void evaluate(Context ctx) {
+
+    auto dev = getValue<input_DEV>(ctx);
+
+    if (isSettingUp())
+        emitValue<output_DEVU0027>(ctx, dev);
+
+    if (isInputDirty<input_DEV>(ctx)) {
+        dev->initTypeB();
```

Comment:
This should be done in the `isSettingUp` branch. Do not check `input_DEV` dirtieness.